ui/pages/permissions-connect/snap-install/index.scss

```diff
@@ -0,0 +1,47 @@
+.snap-install {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+  box-shadow: none;
+  border: none;
+
+  .snap-requests-permission {
+    @include H7;
+
+    padding: 16px;
+  }
+
+  .permissions-connect-permission-list {
+    padding-left: 24px;
+    padding-right: 24px;
```

Comment:
Do you know if we have padding top or bottom?  If not, we can do `padding: 0 24px;`